mlir/include/mlir/IR/BuiltinAttributes.td

```diff
@@ -64,6 +64,7 @@ def Builtin_AffineMapAttr : Builtin_Attr<"AffineMap", [
     AffineMap getAffineMap() const { return getValue(); }
   }];
   let skipDefaultBuilders = 1;
+  let attrName = "builtin.affine_map";
```

Comment:
So I'm just curious - why can't the same reparameterization that's done for `Builtin_Type` (taking the mnemonic as an arg) be done for `Builtin_Attr`?

Reply:
Oh yes, that's an overlook on my side, I fixed it now to work like `Builtin_Type`!